platforms/cpuinfo.go

```diff
@@ -94,6 +94,19 @@ func getCPUVariant() string {
 
 		return variant
 	}
+	if runtime.GOOS == "freebsd" {
+		// FreeBSD supports ARMv6 and ARMv7 as well as ARMv4 and ARMv5 (though deprecated)
+		// I don't know how to detect the variant so let's just return "unknown"
```

Comment:
(nit) I don't think we usually use first-person in comments here.  Maybe write something like "detecting those variants is currently unimplemented"?

Reply:
Done